src/presentation/components/common/FormItem/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode } from 'react';
+
+import { StErrorMsg, StFormItem, StInputStatus, StCount } from './style';
+
+interface FormItemProps {
+  children: ReactNode;
+  value: string;
+  errorMsg?: string;
+  maxLength?: number;
+}
+
+export default function FormItem(props: FormItemProps) {
+  const { errorMsg, children, value, maxLength } = props;
+  return (
+    <StFormItem>
+      {children}
+      <StInputStatus>
+        <StErrorMsg>{errorMsg && errorMsg}</StErrorMsg>
+        {maxLength && (
+          <StCount isMax={maxLength < value.length}>{`${value.length}/${maxLength}`}</StCount>
```

Comment:
별거 아니긴 한데
`const valueLength = value.length;` 하나 추가하고 아래처럼 바꿔도 좋을 것 같습니다!
```suggestion
          <StCount isMax={maxLength < valueLength}>{`${valueLength}/${maxLength}`}</StCount>
```

Reply:
혹시 무슨 차이가 있을까요?? 코드만 한 줄 늘어나는 것은 아닐까용?

Reply:
헉 그렇네요 피드백 감사합니다 !!